File: algorithms/python3/268_missing_number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        res=0
        for i in range(len(nums)):
            res=res+i+1-nums[i]
        return res

a=Solution()

print(a.missingNumber([9,6,4,2,3,5,7,0,1]))

File: algorithms/cpp/268_missing_number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        if (nums.empty()) return 0;
        int diff=0;
        for (int i=0;i<nums.size();i++) 
            diff+=(nums[i]-i);
        return nums.size()-diff;
        }
};
